backup_and_restore/application_backup_and_restore/oadp-performance/oadp-recommended-network-settings.adoc

```diff
@@ -0,0 +1,25 @@
+:_mod-docs-content-type: ASSEMBLY
+[id="oadp-recommended-network-settings.adoc"]
+= {oadp-short} recommended network settings
+include::_attributes/common-attributes.adoc[]
+include::_attributes/attributes-openshift-dedicated.adoc[]
+
+:context: oadp-recommended-network-settings
+
+toc::[]
+
+For a supported experience with {oadp-first}, you should have a stable and resilient network across {OCP-short} nodes, S3 storage, and in supported cloud environments that meet {OCP-short} network requirement recommendations.
+
+To ensure successful backup and restore operations for deployments with remote S3 buckets located off-cluster with suboptimal data paths, Red Hat recommends that your network settings meet the following minimum requirements in such less optimal conditions:
+
+* Bandwidth (network upload speed to object storage): Greater than 2 Mbps for small backups and 10-100 Mbps depending on the data volume for larger backups.
+* Packet loss: 1%
+* Packet corruption: 1%
+* Latency: 100ms
+
+Ensure that your OCP network performs optimally and meets OpenShift network requirements.
+
+[IMPORTANT]
+====
+While Red Hat supports standard backup and restore failures, it does not support failures caused by network settings that do not meet the recommended thresholds.
```

Comment:
Per IBM Style Guide, use _while_ only to refer to time. 
```suggestion
Although Red Hat supports standard backup and restore failures, it does not support failures caused by network settings that do not meet the recommended thresholds.
```
Also, I wonder if _Red Hat supports_ could be worded differently? (I don't think RH supports failures; we don't have much of choice, do we?)
```suggestion
Although Red Hat provides supports for standard backup and restore failures, it does not provide support for failures caused by network settings that do not meet the recommended thresholds.
```